python/python/bystro/vcf_utils/tests/test_sim_data_generator.py

```diff
@@ -0,0 +1,35 @@
+"""Test simulated data."""
+
+import pandas as pd
+from bystro.vcf_utils.sim_data_generator import (
+    generate_random_vcf_index, 
+    generate_simulated_vcf
+)
+
+HEADER_COLS = ["#CHROM", "POS", "ID", "REF", "ALT", "QUAL", "FILTER", "INFO", "FORMAT"]
+
+
+def test_generate_random_vcf_index():
+    """Test expected index components have correct parameters."""
+    random_vcf_index = generate_random_vcf_index()
+    assert isinstance(random_vcf_index, str)
+    parts = random_vcf_index.split(":")
```

Comment:
could write this as something like `chrom, pos, ref, alt = random_vcf_index.split(":")` to make assertions below more readable